contentful/extensions/ad-tags/index.html

```diff
@@ -0,0 +1,98 @@
:#fff2f1}input.cf-form-input:disabled,input.cf-form-input[aria-disabled=true],input.cf-form-input[disabled],select.cf-form-input:disabled,select.cf-form-input[aria-disabled=true],select.cf-form-input[disabled],textarea.cf-form-input:disabled,textarea.cf-form-input[aria-disabled=true],textarea.cf-form-input[disabled]{background-color:#f7f9fa;color:#a9b9c0;-webkit-border-image:none;-moz-border-image:none;-o-border-image:none;border-image:none}input.cf-form-input[size]{width:auto}select.cf-form-input,textarea.cf-form-input{font-size:16px;color:#2a3039;font-family:"Avenir Next W01",Helvetica,sans-serif;display:block;line-height:1.5;outline:0;background-color:transparent;border-width:0 0 1px;border-style:solid;border-color:transparent;border-image-width:0 0 1px 0;border-image-source:url(https://static.contentful.com/app/svg/dotted-border.svg);border-image-repeat:round;border-image-slice:1 1;-webkit-transition:.2s linear;-moz-transition:.2s linear;-o-transition:.2s linear;-ms-transition:.2s linear;transition:.2s linear;-webkit-transition-property:padding,background-color;-moz-transition-property:padding,background-color;-o-transition-property:padding,background-color;-ms-transition-property:padding,background-color;transition-property:padding,background-color}textarea.cf-form-input{max-width:100%;width:100%;padding:.2em 2px 0;height:4.5em}select.cf-form-input{cursor:inherit;background:0 0;border:0;-webkit-appearance:none;-moz-appearance:none;appearance:none;width:auto;padding:0 1.5em 0 .5em;height:36px;background-image:url(https://static.contentful.com/app/svg/dd-arrow-down.svg);background-position:center right 5px;background-repeat:no-repeat;-webkit-background-size:.7em;-moz-background-size:.7em;background-size:.7em}select.cf-form-input::-ms-expand{display:none}select.cf-form-input>option{-o-text-overflow:ellipsis;text-overflow:ellipsis;overflow:hidden}.cf-field-error{margin-top:.642857142857143em;color:#f05650}.cf-form-option{margin-bottom:1.285714285714286em;display:-webkit-box;display:-moz-box;display:-webkit-flex;display:-ms-flexbox;display:box;display:flex}.cf-form-option:last-child{margin-bottom:0}.cf-form-option>input[type=checkbox],.cf-form-option>input[type=radio]{-webkit-align-self:flex-start;align-self:flex-start;-ms-flex-item-align:start;-webkit-flex-shrink:0;flex-shrink:0;margin:1px .5em 0 0}.cf-form-option>label{display:block;cursor:pointer}.cf-form-label p{font-size:13px;margin:0}.cf-form-label label{line-height:1.928571428571429em;margin-top:-.642857142857143em;display:inline-block;cursor:pointer}.cf-form-option>input[type=checkbox]:disabled+label{color:gray}.cf-form-horizontal{display:-webkit-box;display:-moz-box;display:-webkit-flex;display:-ms-flexbox;display:box;display:flex;-webkit-box-lines:multiple;-moz-box-lines:multiple;-o-box-lines:multiple;-webkit-flex-wrap:wrap;-ms-flex-wrap:wrap;flex-wrap:wrap}.cf-form-horizontal .cf-form-option{margin-right:2.571428571428572em}.cf-form-horizontal .cf-form-option:last-child{margin-right:0}
```

Comment:
I believe these classes are actually loaded for you, also I'm not sure if this is the place for a contentful extension.

https://github.com/remedyhealth/hc-ui-extensions/blob/master/extensions/dateTimeOnPublished/index.html#L4

Although its specifically called `hc-ui-extensions` this is where we are keeping our UI extensions for the time-being.